xonsh/history/base.py

```diff
@@ -103,8 +103,8 @@ def append(self, cmd):
         Parameters
         ----------
         cmd: dict
-            A dict contains informations of a command. It should contains
-            the following keys like ``inp``, ``rtn``, ``ts`` etc.
+            A dict contains information on a command. It should contain
+            the following keys, like ``inp``, ``rtn``, ``ts`` &etc.
```

Comment:
I was thinking about this a bit more and I was wondering what you thought of this text:

> This dict contains information about a command. It should contain the following keys; ``inp``, ``rtn``, ``ts`` and others as needed.